build.zig

```diff
@@ -1,4 +1,6 @@
 const std = @import("std");
+const log = std.log.scoped(.zbench_build);
+
 const version = std.SemanticVersion{ .major = 0, .minor = 1, .patch = 0 };
```

Comment:
besides specifying zBench's version, we should also specify a (minimum) required Zig version ( #35 )

Reply:
Is that something you can do through `build.zig`? Or do you mean just in the README?

Reply:
you can check the compiler version and throw @compileError if that doesn't satisfy your needs. Here's an example [in the zls repo](https://github.com/zigtools/zls/blob/master/build.zig#L10).